packages/trpc/server/password.ts

```diff
@@ -0,0 +1,12 @@
+import { z } from 'zod';
```

Comment:
This doesn't feel like the right place for it but I'm not sure where is. `packages/lib/universal/password.ts` might be an idea? 🤔

Reply:
I think `packages/trpc/server/auth-router/schema.ts` or `packages/lib/types/password`

Reply:
@adithyaakrishna